plugins/modules/monitoring/datadog/datadog_event.py

```diff
@@ -54,6 +54,10 @@
         description:
         - Host name to associate with the event.
         - If not specified, it defaults to the remote system's hostname.
+    api_host:
+        type: str
+        description: ["DataDog API endpoint URL."]
```

Comment:
It's a lot easier to edit this way:
```suggestion
        description:
        - DataDog API endpoint URL.
```